pyth_observer/calendar.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+EQUITY_OPEN = datetime.time(9, 30, 0)
+EQUITY_CLOSE = datetime.time(16, 0, 0)
```

Comment:
If you're going to do this, doesn't it make sense to use pytz to specify the timezone for when these things happen?